e2e/commands/index.ts

```diff
@@ -1 +1,2 @@
 export * from './patient-operations';
+export * from './visitOperations';
```

Comment:
Use kebab case for file naming @RandilaP 

```suggestion
export * from './visitOperations';

```

Reply:
@anjula-sack  Done